### DISCUSSION TITLE
Is v3.0.0 not yet recommended?

### DISCUSSION THREAD

**Discussion by unitof** [git-lfs/git-lfs#4649] [General] [answered]
I noticed [3.0.0](https://github.com/git-lfs/git-lfs/blob/main/CHANGELOG.md#300-24-sep-2021) was released here on GitHub a few days back, but the download button on https://git-lfs.github.com still points to 2.13.3.

Just curious if there's a reason for that and 3.0.0 is not yet recommended for all, or if that download button just has yet to be updated. Thanks!

**Comment by bk2204** [comment, marked as answer]
The 3.0.0 release failed to build and is incomplete.  We'll be publishing a 3.0.1 soon (probably today or tomorrow) to address that issue and a necessary bug fix.

**Reply by unitof** [reply]
Got it, thanks!
